testing_gateway.go

```diff
@@ -0,0 +1,42 @@
+package braintree
+
+// This action is only available in the sandbox environment.
+type TestingGateway struct {
+	*TransactionGateway
+}
+
+// Settle settles a transaction.
+func (g *TestingGateway) Settle(id string) (*Transaction, error) {
```

Comment:
I reckon given that this is no longer in the context of the transaction gateway, we should name the `id` variable `transactionID` to make it clear to others that's the type of ID we're expecting.